### PR TITLE
fix off by one error when adding bookmark

### PR DESCRIPTION
**Pull request by pascalwacker**

### CODE REVIEW

**Comment by sarstein** [comment]
i wirte down a comment

File: mission3/bookmarkmgr.htm [hunk 1]
```diff
@@ -26,7 +26,7 @@
                     const bookmarks = this.readBookmarks();
 
                     //If the URL is already in the list of bookmarks, or if it is invalid, do not add it.
-                    if (bookmarks.indexOf(url) > 0) {
```
Comment (sarstein) [inline mission3/bookmarkmgr.htm]: hi comment 1

File: mission3/bookmarkmgr.htm [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
+                    if (bookmarks.indexOf(url) >= 0) {
```
Comment (pascalwacker) [inline mission3/bookmarkmgr.htm]: Test

File: mission3/bookmarkmgr.htm (hunk 1 above)
Comment (pascalwacker) [reply mission3/bookmarkmgr.htm]: Hi

Comment (pascalwacker) [inline mission3/bookmarkmgr.htm]: Foobar

File: mission3/bookmarkmgr.htm [hunk 3]
```diff
@@ -26,8 +26,9 @@
...
                         return;
+			// test für sara
```
Comment (sarstein) [inline mission3/bookmarkmgr.htm]: huu

File: mission3/bookmarkmgr.htm (hunk 2 above)
Comment (sarstein) [reply mission3/bookmarkmgr.htm]: test test

**Pull request merged**